packages/platform-browser/src/errors.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+/**
+ * The list of error codes used in runtime code of the `platform-browser` package.
+ * Reserved error code range: 5000-5500.
+ */
+export const enum RuntimeErrorCode {
+  BROWER_MODULE_ALREADY_LOADED = 5000,
+  NO_PLUGIN_FOR_EVENT = 5001,
+  UNSUPPORTED_EVENT_TARGET = 5002,
+  TESTABILITY_NOT_FOUND = 5003,
+  ROOT_SELECTOR_NO_MATCH = 5004,
+  UNEXPECTED_SYNTHETIC = 5005,
+
+  SANITIZATION_UNSAFE_SCRIPT = 5006,
+  SANITIZATION_UNSAFE_RESOURCE_URL = 5007,
+  SANITIZATION_UNEXPECTED_CTX = 5008,
```

Comment:
We can probably have a sub-range here:
```suggestion
  // Sanitization-related errors (5100-5200 range)
  SANITIZATION_UNSAFE_SCRIPT = 5100,
  SANITIZATION_UNSAFE_RESOURCE_URL = 5101,
  SANITIZATION_UNEXPECTED_CTX = 5102,
```